commands/install_license

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
+help() {
+  cat <<EOF
+----------------------------------------------------------------------------------------------------------
+HELP MESSAGE:
+
+    Usage: c8y oee $(basename "$0") [--license <license-string>] [--filePath <path-to-license-file>]
+    Arguments:
+      -l, --license   license-string        The license string
+      -f, --filePath  path-to-license-file  The path to the xml license file
+    ***NOTE*** One of these arguments must be used !!!
+
+    Example usage:
+      Install Apama license via c8y tenant option by input license string in command line
+          c8y oee install_license --license 'ABCxyz-key-welcome-to-OEE'
+      **Note** :
+        1. The license string must be covered by single quote, i.e '<license-string>'.
+        2. Any new line character must be removed to let terminal recognize the license string as a single string.
+
+      Install Apama license via c8y tenant option by reading xml license file in any directory:
+          c8y oee install_license --filePath "C:/path/to/your/license/file/license_file.xml"
+
+      Install Apama license via c8y tenant option by reading xml license file in c8y-oee/licenses:
+          c8y oee install_license
+          -> user is provided a list of files to choose.
+
+----------------------------------------------------------------------------------------------------------
+EOF
+}
+
+error_check() {
+  if [ $? -ne 0 ]; then
+    echo "Error: process failed"
+    exit 1
+  fi
+}
+
+
+# Parse command-line options
+REST_ARGS=()
+while [ $# -gt 0 ]; do
+  case "$1" in
+    -h|--help)
+      help
+      exit 0
+      ;;
+    -l|-license|--license)
+      if [ $# -gt 1 ]; then
+        LICENSE_UNESCAPED=$2
+        shift
+      else
+        echo "Option $1 requires an argument." >&2
+        exit 1
+      fi
+      ;;
+    -f|-filePath|--filePath)
+      if [ $# -gt 1 ]; then
+        FILEPATH=$2
+        shift
+      else
+        echo "Option $1 requires an argument." >&2
+        exit 1
+      fi
+      ;;
+    --*)
+      echo "Invalid option: $1"
+      help
+      exit 0
```

Comment:
with the latest version I'm getting an error when trying to use "--dry" or "--debug"; maybe it is related to this change?

```
PS C:\Users\mvoigt> c8y oee-cli.naph  install_license --dry --license '<?xml version="1.0" encoding="US-ASCII"?><SoftwareAG_License><Component Id="SalesInfo"><SerialNumber>0000466511</SerialNumber><LicenseKey>A76FC7B9AC3D29DC4207F0C4C7EBB8BD</LicenseKey><CustomerID>3150000</CustomerID><CustomerName>Software AG for Solution Book Internal USE ONLY</CustomerName><ContractDetails>SOFTWARE AG internal license solely for sales promotion and internal training purposes. Please note that this access key is for your personal use only. Access and software shall not be forwarded to anybody else</ContractDetails><ContractPosNumber/><LicenseTypeDetails>Internal</LicenseTypeDetails><AutoCallHome>No</AutoCallHome></Component><Component Id="ProductInfo"><ExpirationDate>2023/10/31</ExpirationDate><OS>Linux</OS><ProductCode>PAQ</ProductCode><ProductName>Apama for Cumulocity IoT</ProductName><ProductVersion>10.1</ProductVersion><Usage/><RenewalDate/><HigherLevPosNr/><HigherLevProdCode/><HigherLevProdName/></Component><Component Id="LicenseInfo"><PriceUnit>SCP</PriceUnit><PriceQuantity>1</PriceQuantity><TypeOfInstallation/><LicenseType/><LicenseVersion>1.2</LicenseVersion><Metering>MNO</Metering><ExtendedRights/></Component><Component Id="ApamaServer"><ProductCode>PAM</ProductCode><ProductVersion>10.1</ProductVersion><MaxResidentMemoryMB>Unlimited</MaxResidentMemoryMB></Component></SoftwareAG_License>'
Invalid option: --dry
----------------------------------------------------------------------------------------------------------
HELP MESSAGE:

    Usage: c8y oee install_license [--license <license-string>] [--filePath <path-to-license-file>]
    Arguments:
      -l, --license   license-string        The license string
      -f, --filePath  path-to-license-file  The path to the xml license file
    ***NOTE*** One of these arguments must be used !!!

    Example usage:
      Install Apama license via c8y tenant option by input license string in command line
          c8y oee install_license --license 'ABCxyz-key-welcome-to-OEE'
      **Note** :
        1. The license string must be covered by single quote, i.e '<license-string>'.
        2. Any new line character must be removed to let terminal recognize the license string as a single string.

      Install Apama license via c8y tenant option by reading xml license file in any directory:
          c8y oee install_license --filePath "C:/path/to/your/license/file/license_file.xml"

      Install Apama license via c8y tenant option by reading xml license file in c8y-oee/licenses:
          c8y oee install_license
          -> user is provided a list of files to choose.

----------------------------------------------------------------------------------------------------------
```